src/OpenSage.Game/Assets/Shaders/Sprite.frag

```diff
@@ -62,5 +62,11 @@ void main()
 
     textureColor *= in_Color;
 
+    if (_SpriteConstants.Grayscale)
+    {
+        float gray = 0.299 * textureColor.r + 0.587 * textureColor.g + 0.114 * textureColor.b;
```

Comment:
There are multiple formulas to convert color to grayscale, maybe we should document which one we use?